### ISSUE TITLE
Tab: Paying Contacts > Contacts

### ISSUE THREAD

**Issue by mcharg** [ONRR/onrr.gov-site#1531]
https://qmcq5k.axshare.com/#id=rmsbze&p=payments&g=1

**Comment by mcharg** [comment]
@cthomasONRR This is ready to review.

**Comment by cthomasONRR** [comment]
@mcharg This was almost impossible to review. I think the wireframes are now out of date but here are inconsistencies I found:

**Federal accounts receivable**
- Solid Minerals is the first listing in dev, not it alphabetical order.
- In dev, John O'Donoghue's name is giving a character error:
![image](https://user-images.githubusercontent.com/72526208/162753037-7b4c8204-fc7e-4efd-972a-ef6a5fc2e8dc.png)
- Entries where dev does not match the wireframe: Companies starting with: DEE-ENERGYX, ENERGYQ-EXXE, EXXO-GIF, GIL-HUE, HUF-LIA, MARU-NE, PEAK-RED, SCHL-SS, ST-TEXI, TEXL-WALD, WALE-Z, Geothermal, Civil Penalites, Inspection fees, Refunds
- Possible typo: is it "EXXO-GIF" or "EXXO-GIB" not sure which is correct? Same for "REE-SCHIF" or "REE-SCHA" 

**Debt collection**
-  Entries where dev does not match the wireframe: Companies starting with: E, T, Q; Solids: MA006-463

**Comment by cthomasONRR** [comment]
@mcharg Names check out now.

There still needs to be reorganization. "Solid Minerals" should not be the very first entry fi they are supposed to be in alphabetical order....
![image](https://user-images.githubusercontent.com/72526208/162769864-7b40a719-980e-4a30-bf3b-d88cb1753ad0.png)

**Comment by mcharg** [comment]
@cthomasONRR I created a new issue for @jlouis-onrr to look at sorting.

**Comment by mcharg** [comment]
@cthomasONRR Created #1826 for the parentheses.

**Comment by cthomasONRR** [comment]
@mcharg Jerome fixed the card sorting, so that's good on this page now :) Just waiting on apostrophes.

**Comment by cthomasONRR** [comment]
@mcharg I set this to approved in the spreadsheet since there's an issue out for the apostrophes
